images/dask/README.md

```diff
@@ -1,13 +1,25 @@
-# dask image
+# dask Image
 
-This image exists to add necessary dependencies for dask schedulers
+This image exists to add necessary dependencies for Dask schedulers
 and workers, such as the Prometheus client and Python libraries
-used by analytics code. See [the dask docs](https://docs.dask.org/en/stable/how-to/manage-environments.html)
+used by analytics code. See [the Dask docs](https://docs.dask.org/en/stable/how-to/manage-environments.html)
 for some additional detail.
 
-## Building and pushing manually
+## Testing Changes
+
+A person with Docker set up locally can build a new version of the Dask image at any time after making changes.
+
+```bash
+docker build -t ghcr.io/cal-itp/data-infra/dask:2022.10.13 .
+```
+
+## Deploying Changes to Production
+
+When changes are finalized, a new version number should be specified in [version.txt](./version.txt). When changes to this directory are merged into `main`, [a GitHub Action](../../.github/workflows/build-dask-image.yml) automatically publishes an updated version of the Dask image. A contributor with proper GHCR permissions can also manually deploy a new version of the image via the CLI:
 
 ```bash
 docker build -t ghcr.io/cal-itp/data-infra/dask:2022.10.13 .
```

Comment:
Do we want to specify that you would list your new version number in these commands? I guess it's implied but may be good to be specific

Reply:
Resolved in [696ac34](https://github.com/cal-itp/data-infra/pull/3156/commits/696ac3432bae8a47626ed6bf35dc191c0254d2b9)